release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,21 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-column-management"]
+==== Column management
```

Comment:
Can this be more specific, for example "Column management on resources pages" or "Resources pages column management"?